29-restroom-servicing-monitor/README.md

```diff
@@ -0,0 +1,98 @@
+  - [Blues Community](#blues-community)
+  - [Additional Resources](#additional-resources)
+
+## You Will Need
+
+* [Sparrow Development Kit](https://shop.blues.io/products/sparrow-dev-kit)
+* 2 USB A to micro USB cables
+* Breadboard
+* [3 push buttons](https://www.sparkfun.com/products/14460)
+* Jumper wires
+
+## Notehub Setup
+
+Sign up for a free account on [notehub.io](https://notehub.io) and [create a new project](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-pi/#set-up-notehub).
+
+## Sparrow Setup
+
+### Quickstart
+
+Follow the [Sparrow Quickstart](https://dev.blues.io/quickstart/sparrow-quickstart/) to get your Sparrow devices paired with each other and associated with the Notehub project you just created. Make sure that you use the ProductUID generated in [Notehub Setup](#notehub-setup) when it comes time to issue the `hub.set` command in the quickstart. Note also that we'll only need one reference node for this project, so you don't need to pair both nodes that came with the dev kit.
+
+After you've completed the quickstart, leave the Notecarrier and essentials board powered and connected. These two devices will act as our gateway to Notehub, and we won't need to touch them again. The rest of this guide will focus on the reference node.
+
+### Hardware
+
+Before we set up the custom firmware needed for the restroom servicing monitor, let's wire up the hardware. The hardware interface is 3 push buttons: 1 for a bad rating (restroom needs service), 1 for an ok rating (restroom cleanliness deteriorating), and 1 for a good rating (no service needed). These buttons are wired to GPIO pins on the reference node. Those pins use internal pull-ups so that the pin reads a digital 1 when the button isn't pushed. When the button is pushed, it connects the corresponding GPIO pin to GND so that it reads a digital 0. Using the breadboard, buttons, and jumper wires, wire up the hardware as shown in the diagram below.
+
+![breadboard wiring](assets/nf29_breadboard.jpg "Breadboard Wiring")
+
+The labels off the breadboard (A1-A3 and GND) correspond to pins on the reference node. Connect these to the breadboard as shown with jumper wires.
+
+### Firmware
+
+Next, we need to flash the reference node with the restroom servicing monitor firmware.
+
+1. Before we do anything, we need to pull in some dependencies for the firmware to work. After cloning this repository, run `git submodule update --init`. This will pull in the note-c and sparrow-lora submodules that this project depends on.
+1. Install [Visual Studio Code](https://code.visualstudio.com/). We will program the reference node from VS Code.
+1. Add the [Dev Containers extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) to VS Code. Building and flashing the firmware will take place inside a Docker container specifically designed for these tasks (refer to firmware/Dockerfile for details of what's in the container).
```

Comment:
Docker is very Linux-centric, you make want to leave tool installation to the Quickstart.

Reply:
Added a commit to make these docs OS/IDE agnostic by deferring those bits to the Sparrow Builder's Guide.